components/layout_2020/flow/inline/mod.rs

```diff
@@ -172,8 +172,22 @@ pub(crate) struct InlineFormattingContext {
 /// A collection of data used to cache [`FontMetrics`] in the [`InlineFormattingContext`]
 #[derive(Debug)]
 pub(crate) struct FontKeyAndMetrics {
+    // Font instance key allows us to get font from system
+    // However this font doesn't contain information that
+    // stored in font_descriptor, only about template.descriptor.
+    // Nevertheless it seems that
+    // created keys would be unique for the font files.
```

Comment:
What information is missing exactly? I think the bug here is that `FontInstanceKey` isn't identifying the font uniquely, but this change just papers around the issue.